tests/Microsoft.Graph.DotnetCore.Core.Test/Mocks/MockUserEventsCollectionRequest.cs

```diff
@@ -0,0 +1,98 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using Microsoft.Graph.DotnetCore.Core.Test.TestModels.ServiceModels;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Graph.DotnetCore.Core.Test.Requests;
+
+namespace Microsoft.Graph.DotnetCore.Test.Tasks
+{
+    public class MockUserEventsCollectionRequest : TestEventDeltaRequest
+    {
+        private TestEventDeltaCollectionPage nextPage;
```

Comment:
could we replace this class by a mocking framework?

Reply:
Will investigate this.